docs/040_pepr-tutorials/040_pepr-gitops-workflow.md

```diff
@@ -0,0 +1,5 @@
+# Using Pepr in a GitOps Workflow
+
+Admission Controllers and Operators can be complimentary to GitOps workflows. The important part is to ensure that the GitOps workflow is not disrupted by the admission controller or operator. This can be achieved by ensuring that the admission controller or operator does not modify the resources in a way that would conflict with the GitOps workflow.
```

Comment:
```suggestion
Admission Controllers and Operators can be complimentary to GitOps workflows. The important part is to ensure that the GitOps workflow is not disrupted by the admission controller or operator. This can be achieved by clearly defining the boundaries in the GitOps Server in configuration.
```